crates/anonymization/src/core/word.rs

```diff
@@ -0,0 +1,92 @@
+use std::collections::{HashMap, HashSet};
+
+use cosmian_crypto_core::CsRng;
+use rand::{RngCore, SeedableRng};
+use regex::Regex;
+
+use super::AnoError;
+
+pub struct WordTokenizer {
+    word_token_mapping: HashMap<String, [u8; 16]>,
+    re_word_separator: Regex,
+}
```

Comment:
it was arbitrarily set to the same length as a UUIDv4.
Const generic cannot be exposed in the other languages interfaces so it will need to be hard-coded to 16 bytes at some point anyway.